ui/_locales/en/messages.json

```
@@ -214,8 +248,56 @@
     "close": "Close",
     "accountItemSummary": {
       "connectedStatus": "Connected"
+    },
+    "selectToken": "Select token",
+    "backButtonText": "Back",
+    "saveBtn": "Save"
+  },
+  "signing": {
+    "signatureRequired": "Your signature is required",
+    "dappSignatureRequest": "A dapp is requesting your signature",
+    "message": "Message",
+    "signed": "Signed,",
+    "type": "Type",
+    "EIP4361": {
+      "subtext1": "Wants you to sign in with your",
+      "subtext2": "Ethereum account:",
+      "statement": "Statement",
+      "nonce": "Nonce",
+      "version": "Version",
+      "chainID": "Chain ID",
+      "expiration": "Expiration"
     }
   },
+  "signTransaction": {
+    "awaitingHardwareSignature": "Awaiting hardware wallet signature",
+    "cannotRejectWarning": "Tx can only be Rejected from Ledger",
+    "reject": "Reject",
+    "checkLedger": "Check Ledger",
+    "noSigning": "Read-only accounts cannot sign",
+    "sendTo": "Send to",
+    "contractCreation": "Contract creation",
+    "interactingWith": "Interacting with",
+    "unknownNetwork": "Unknown Network",
+    "detailPanelName": "Details",
+    "rawDataPanelName": "Raw data",
+    "rawDataCopyMsg": "Raw data copied to clipboard",
+    "copyRawData":  "Copy hex",
+    "spendAmount": "Spend Amount",
+    "help": "I need help",
+    "refresh": "Refresh",
+    "spendAsset": {
+      "approve": "Approve",
+      "infinite": "Infinite",
+      "spend": "Spend",
+      "tokens": "tokens",
+      "spendLimit": "Spend limit",
+      "tooltip1": "Spend limit is the amount of funds from a particular asset, that you allow a contract to spend.",
+      "tooltip2": "Infinite tx has the drawback that if the contract is mailicous, it can steal all your funds.",
```

```suggestion
      "tooltip2": "Infinite tx has the drawback that if the contract is malicous, it can steal all your funds.",
```

```suggestion
      "tooltip2": "Infinite tx has the drawback that if the contract is malicous, it can steal all your funds.",
```

While we're in here...

---

addressed in https://github.com/tallycash/extension/pull/2289